src/current/_includes/metrics-usage/sql.ddl.count.md

```diff
@@ -1 +1 @@
-This high-level metric reflects workload volume. Monitor this metric to identify abnormal application behavior or patterns over time. If abnormal patterns emerge, apply the metric's time range to the <a href="https://www.cockroachlabs.com/docs/stable/monitoring-and-alerting#sql-activity-pages"><b>SQL Activity</b> pages</a> to investigate interesting outliers or patterns. For example, on the <a href="https://www.cockroachlabs.com/docs/cockroachcloud/transactions-page"><b>Transactions</b> page</a> and the <a href="https://www.cockroachlabs.com/docs/cockroachcloud/statements-page"><b>Statements</b> page</a>, sort by the <b>Execution Count</b> column. To find problematic sessions, on the <a href="https://www.cockroachlabs.com/docs/cockroachcloud/transactions-page"><b>Sessions</b> page</a>, sort by the <b>Transaction Count</b> column. Find the sessions with high transaction counts and trace back to a user or application.
\ No newline at end of file
+This high-level metric reflects workload volume. Monitor this metric to identify abnormal application behavior or patterns over time. If abnormal patterns emerge, apply the metric's time range to the [<b>SQL Activity</b> pages]({% link {{ site.current_cloud_version }}/monitoring-and-alerting.md %}#sql-activity-pages) to investigate interesting outliers or patterns. For example, on the [<b>Transactions</b> page]({% link cockroachcloud/transactions-page.md %}) and the [<b>Statements</b> page]({% link cockroachcloud/statements-page.md %}), sort by the <b>Execution Count</b> column. To find problematic sessions, on the [<b>Sessions</b> page]({% link cockroachcloud/sessions-page.md %}), sort by the <b>Transaction Count</b> column. Find the sessions with high transaction counts and trace back to a user or application.
```

Comment:
I feel the `({% link {{ site.current_cloud_version }}/monitoring-and-alerting.md %}#sql-activity-pages)` links are confusing because they go to the Self-Hosted / DB Console doc, rather than to a corresponding Cloud doc (I know we don't have one). IMO we can do without the link on `SQL Activity pages`, especially because we have the direct Transactions and Statements page links. Wdyt?